examples/with-docker/Dockerfile

```diff
@@ -3,7 +3,8 @@ FROM node:16-alpine AS deps
 # Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
 RUN apk add --no-cache libc6-compat
 WORKDIR /app
-COPY package.json yarn.lock ./
+# replace with "COPY package.json yarn.lock ./" if you have yarn.lock file
+COPY package.json ./ 
```

Comment:
```suggestion
# Replace yarn.lock with package-lock.json if using npm
COPY package.json yarn.lock ./ 
```

Yeah so I would say let's keep the comment here, but don't remove `yarn.lock`.

Reply:
If you use 
`npx create-next-app --example with-docker docker-app`,
you will need this command `COPY package.json package-lock.json ./`,
but if you use 
`yarn create next-app --example with-docker docker-app`,
you will need this command `COPY package.json yarn.lock ./`

Reply:
update: found how to solve this

Reply:
I've changed it, so it automatically copy `package-lock.json` or `yarn.lock` and doesn't fail when these files are not found